sirepo/template/ml.py

```diff
@@ -301,13 +301,26 @@ def stateless_compute_remote_data_bytes_loaded(data):
     return _remote_data_bytes_loaded(data.filename)
 
 
+def stateless_compute_get_archive_file_list(data):
+    return _archive_file_list(data.filename, data.data_type)
+
+
 def write_parameters(data, run_dir, is_parallel):
     pkio.write_text(
         run_dir.join(template_common.PARAMETERS_PYTHON_FILE),
         _generate_parameters_file(data),
     )
 
 
+def _archive_file_list(filename, data_type):
+    def _filter(item):
+        is_dir = getattr(item, a.dir_check)()
```

Comment:
Single letter variables aren't used outside scope. Pass it into _filter or name it and define it above _filter.

Reply:
[4871694](https://github.com/radiasoft/sirepo/pull/4736/commits/4871694b7b14e7a7e255c3872bdfae5afe91af43)